packages/perseus/src/strings.ts

```diff
@@ -675,133 +673,434 @@ export const strings = {
+    },
+    srLinearSystemPoints: {
+        context:
+            "Screen reader description for the points of a line in the Linear System in the interactive graph widget.",
+        message:
+            "Line %(lineNumber)s has two points, point 1 at %(point1X)s comma %(point1Y)s and point 2 at %(point2X)s comma %(point2Y)s.",
+    },
+    srLinearSystemPoint: {
+        context:
+            "Screen reader description for a point on a line in the Linear System in the interactive graph widget.",
+        message:
+            "Point %(pointSequence)s on line %(lineNumber)s at %(x)s comma %(y)s.",
+    },
+    srLinearSystemGrabHandle: {
+        context:
+            "Aria label for the interactive segment that allows the user to move a whole line in the Linear System in the interactive graph widget.",
+        message:
+            "Line %(lineNumber)s going through point %(point1X)s comma %(point1Y)s and point %(point2X)s comma %(point2Y)s.",
+    },
+    srLinearSystemIntersection: {
+        context:
+            "Screen reader description for the intersection of two lines in the Linear System in the interactive graph widget.",
+        message: "Line 1 and line 2 intersect at point %(x)s comma %(y)s.",
+    },
+    srLinearSystemParallel: {
+        context:
+            "Screen reader description when two lines are parallel in the Linear System in the interactive graph widget.",
+        message: "Line 1 and line 2 are parallel.",
+    },
+    srRayGraph: {
+        context:
+            "Screen reader description for the container containing a Ray in the interactive graph widget.",
+        message: "A ray on a coordinate plane.",
+    },
+    srRayPoints: {
+        context:
+            "Screen reader description for the points of a ray in the interactive graph widget.",
+        message:
+            "The endpoint is at %(point1X)s comma %(point1Y)s and the ray goes through point %(point2X)s comma %(point2Y)s.",
+    },
+    srRayGrabHandle: {
+        context:
+            "Aria label for the interactive segment that allows the user to move the whole Ray in the interactive graph widget.",
+        message:
+            "Ray with endpoint %(point1X)s comma %(point1Y)s going through point %(point2X)s comma %(point2Y)s.",
+    },
+    srRayEndpoint: {
+        context:
+            "Aria label for the defining point of the Ray in the interactive graph widget that sits on the tail of the Ray.",
+        message: "Endpoint at %(x)s comma %(y)s.",
+    },
+    srRayTerminalPoint: {
+        context:
+            "Aria label for the defining point the Ray goes through in the interactive graph widget that determines the direction of the Ray.",
+        message: "Through point at %(x)s comma %(y)s.",
+    },
+    srQuadraticGraph: {
+        context:
+            "Aria label for the container containing a Quadratic function in the interactive graph widget.",
+        message: "A parabola on a 4-quadrant coordinate plane.",
```

Comment:
```suggestion
        message: "A parabola.",
```

Why mention the number of quadrants shown in the graph? I don't think this is actually correct; e.g. the graph might only show quadrant 1.

Reply:
I think we added "4-quadrant" this time because so many of the related strings refer to the quadrant number that something is in. Even if the vertex doesn't show up on the graph, it'll read the quadrant that it's in.

It's a fair point though. Since this would require copy updates from a designer, I'm going to punt it to the phase 2 string updates.